third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -269,7 +270,8 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
             // Some reference tests create different errors since variable names are
             // known without optimizations, so we need to have a different exp file
             exp_suffix: Some("no-opt.exp"),
-            options: opts.clone().set_experiment(Experiment::OPTIMIZE, false),
+            options: opts.clone().set_experiment(Experiment::OPTIMIZE, false)
+                .set_experiment(Experiment::ACQUIRES_CHECK, false),
```

Comment:
Should we add the same todo as above here (i.e., `// TODO(#13485): Need to turn off acquires check for now to test 2.0 access specifiers`), so that we do not miss turning this back on later?

Reply:
I think its fine, it is just 15 lines below the other comment, and both test configs are rather symmetric and belong together.